website/docs/r/vcenter.html.markdown

```diff
@@ -37,6 +37,9 @@ The following arguments are supported:
 * `refresh_vcenter_on_read` - (Optional) An optional flag to trigger refresh operation on the
   underlying vCenter on every read. This might take some time, but can help to load up new artifacts
   from vCenter (e.g. Supervisors). Update is a no-op. Default `false`.
```

Comment:
I'm thinking whether we should add a note when it is not useful to do so, as it is a heavy operation and it even produces a visible task in UI. Or at least mention that this produces that task, so users know where it is coming from.

Reply:
Made an attempt to rephrase both - this and `refresh_policies_on_read`